tests/QARegressionTests/test_Issues/test_CoreIssue164.py

```diff
@@ -0,0 +1,119 @@
+import subprocess
+import os
+import openpyxl
+import pytest
+
+
+@pytest.mark.regression
+def test_negative_dataset():
+    command = (
+        "python3 -m core test -s sdtmig -v 3.4 -r "
```

Comment:
This command breaks when running in a virtual env, because it is using the global instance of python instead of the venv instance of python that has the requirements installed.

Maybe you can do something like this:
https://stackoverflow.com/questions/749711/how-to-get-the-python-exe-location-programmatically

Reply:
updated